osmaxx/profile/views.py

```diff
@@ -18,14 +18,15 @@
 class SendVerificationEmailMixin(object):
     RATE_LIMIT_SECONDS = 30
 
-    def _send_email_verification(self, profile):
-        if cache.get(profile.associated_user.id):
+    def _send_email_verification(self):
```

Comment:
Do we ensure that the profile exists before calling this function? Before the caller had to pass the profile and take care of this. Maybe using an `assert user.profile` might remedy this issue?